spec/lib/jit_preloader/preloader_spec.rb

```diff
@@ -55,7 +55,7 @@
           end
         end
 
-        contact_queries = [contact1, contact2, contact3].product([["addresses.count", "addresses.maximum", "phone_numbers.count"]])
+        contact_queries = [contact1, contact2, contact3].to_a.product([["addresses.count", "addresses.maximum", "phone_numbers.count"]])
```

Comment:
Oops, yeah, I may have gone overboard on this change. In Rails 5, AR::Relation stopped delegating missing methods to the loaded collection so I had to explicitly convert any scope. I'll dial these back.